### DISCUSSION TITLE
Image Encoding

### DISCUSSION THREAD

**Discussion by MohamedAtta-AI** [microsoft/psi#289] [General]
Hello, I am new to the framework and I noticed that in the 2nd workshop, the function used to encode the image (EncodeJpeg) and the buffer (GetBuffer) can't be found. Is it changed?
I am trying to send video frames to a model implemented in Python. How can I work around this?

**Comment by sandrist** [comment]
Hello, the `EncodeJpeg` operator can be found in both the [Microsoft.Psi.Imaging.Windows](https://github.com/microsoft/psi/blob/d08fdd34f6957a92a6343d6e7978c2b8efc5f83a/Sources/Imaging/Microsoft.Psi.Imaging.Windows/ImagingOperators.cs#L19C55-L19C65) and Microsoft.Psi.Imaging.Linux projects.

This might be a different place than where it was at the time of the workshop, I'm not sure.

**Reply by MohamedAtta-AI** [reply]
Oh, thank you. I was using [Microsoft.Psi.Imaging](https://github.com/microsoft/psi/tree/d08fdd34f6957a92a6343d6e7978c2b8efc5f83a/Sources/Imaging/Microsoft.Psi.Imaging) for both platforms. Now I see.
